mteb/tasks/STS/multilingual/STS17CrosslingualSTS.py

```diff
@@ -67,7 +67,76 @@ class STS17Crosslingual(AbsTaskSTS, MultilingualTask):
 }""",
         descriptive_stats={
             "n_samples": {"test": 500},
-            "avg_character_length": {"test": 43.3},
+            "avg_character_length": {
+                "test": {
+                    "ko-ko": {
```

Comment:
This does become quite nested. Might be nice to have the overall metrics as well:

{"test": {"n_samples": ... "average_sentence1_length": .., "hf_subset_descriptive_stats": {"ar-ar": {...}}}

Just to keep the easy overview.

Reply:
This would also make multilingual datasets have the same structure as non-multilingual datasets (which we def. want)